### PR TITLE
[RM Ch08] Extreme-edge devices reference

### PR DESCRIPTION
**Pull request by joaoarodrigues**

### CODE REVIEW

**Review by walterkozlowski — APPROVED** [review]

**Review by petorre — APPROVED** [review]

**Comment by walterkozlowski** [comment]
At RM Meeting pn 23 November 2022: discussion that we do not cover orchestration in RM; discussion to be continued post Nile

**Comment by joaoarodrigues** [comment]
The main point here was to consider extreme-edge devices, its not about the orchestration

File: doc/ref_model/chapters/chapter08.rst [hunk 1]
```diff
@@ -659,6 +659,8 @@ operations and management complexity.
 At the edge, the infrastructure may further include ruggedised hardware for harsh environments and hardware with
 different form factors.
 
+Finally, there is a need to integrate, and to consider extreme-edge devices which can be part of the ecosystem, and offer programability and processing capabilities. Ultimately, can be considered for the end-to-end orchestration, and can be used to comply with extreme requirements for some of the use cases.
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter08.rst:662]: Please wrap lines to 120 chars.

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: doc/ref_model/chapters/chapter08.rst (hunk 1 above)
Comment (joaoarodrigues) [inline doc/ref_model/chapters/chapter08.rst:662]:
```suggestion
Finally, there is a need to integrate, and to consider extreme-edge devices which can be part of the ecosystem,
 and offer programability and processing capabilities. Ultimately, can be considered for the end-to-end 
 orchestration, and can be used to comply with extreme requirements for some of the use cases. 
```

**Review by CsatariGergely — APPROVED** [review]

**Review by rgstori — APPROVED** [review]

**Comment by jbhartley** [comment]
Suggest:

1. We include content into prior paragraph
2. Clarify wording

Suggest:

At the edge, the infrastructure may further include ruggedised hardware for harsh environments and hardware with
different form factors. With the evolution of IoT and ubiquitous connectivety (inclusive of personal devices) to consider extreme-edge devices as part of the ecosystem.  This will require need to integrate with and offer programability and processing capabilities for these devices. The end-to-end orchestration will need to support the extreme edge use cases.

File: doc/ref_model/chapters/chapter08.rst [hunk 2]
```diff
@@ -659,6 +659,10 @@ operations and management complexity.
 At the edge, the infrastructure may further include ruggedised hardware for harsh environments and hardware with
 different form factors.
 
+Finally, there is a need to integrate, and to consider extreme-edge devices which can be part of the ecosystem,
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter08.rst:662]: As discussed in the RM meeting of 2022-12-07, starting the sentence with "Finally" doesn't follow the previous sentences. If this suggestion is accepted, then lines 663 and 664 should be deleted. Orchestration is not the topic of this section and so is being removed.

If orchestration must be mentioned, then it should be a separate paragraph that applies to the content of this section.
```suggestion
Additionally, the extreme edge may consist of programmable devices as part of the
Telco Edge Cloud ecosystem for use in meeting some extremely low latency requirements,
```

**Comment by steelescotr** [comment]
@walterkozlowski - would you be able to finalize this PR so we can be sure it's in the Nile Release?

**Comment by joaoarodrigues** [comment]
> Suggest:
> 
> 1. We include content into prior paragraph
> 2. Clarify wording
> 
> Suggest:
> 
> At the edge, the infrastructure may further include ruggedised hardware for harsh environments and hardware with different form factors. With the evolution of IoT and ubiquitous connectivety (inclusive of personal devices) to consider extreme-edge devices as part of the ecosystem. This will require need to integrate with and offer programability and processing capabilities for these devices. The end-to-end orchestration will need to support the extreme edge use cases.

@jbhartley Thank you for the proposal. I've updated it, and corrected a couple of typo's 😊

**Comment by joaoarodrigues** [comment]
> @walterkozlowski - would you be able to finalize this PR so we can be sure it's in the Nile Release?

**Pull request merged**